### PR TITLE
Update release notes for formatting, content, add 6.17.0 release note

### PR DESCRIPTION
**Pull request by dneary**
### What does this PR do?

Modify release notes with some edits for style and content, and ensure consistent file naming for release note files.

### CODE REVIEW

**Comment by dneary** [comment]
Can someone other than me review and approve this merge, please?

**Comment by dneary** [comment]
I see what happened here - I used master rather than a branch for the PR, so when I added a commit for 6.17.0 RN update, it added it here. Is this good to merge like this?

**Review by ghost — APPROVED** [review]

File: release-notes/release-6.15.0.md [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
+The Eclipse Che 6.15.0 release contains the following notable features:
+ 
 * **New CORS configuration parameters**: Allowing to test new CORS configuration before Che 6.16 release
 * **Prometheus metrics endpoint**: Allowing to expose http endpoint with different metrics in prometheus format
 * **More detailed tracing of Workspace operations**: Allowing for more detailed performance monitoring
 * **Jaeger deployed in the Helm chart**: Allowing to collect tracing data in Kubernetes environment
 
+---
+
+This release of Eclipse Che comes with significant improvements in monitoring and tracing capabilities for both the Che server and for developer workspaces. There are also a large number of incremental improvements
```
Comment (rkratky) [inline release-notes/release-6.15.0.md]:
```suggestion
This release of Eclipse Che comes with significant improvements in monitoring and tracing capabilities for both the Che server and for developer workspaces. There are also a large number of incremental improvements.
```

File: release-notes/release-6.15.0.md [hunk 2]
```diff
@@ -1,16 +1,39 @@
-## Release summary
-
-Eclipse Che 6.15 includes:
+# Eclipse Che 6.15 Release Notes
 
+The Eclipse Che 6.15.0 release contains the following notable features:
+ 
 * **New CORS configuration parameters**: Allowing to test new CORS configuration before Che 6.16 release
 * **Prometheus metrics endpoint**: Allowing to expose http endpoint with different metrics in prometheus format
```
Comment (rkratky) [inline release-notes/release-6.15.0.md]:
```suggestion
* **Prometheus metrics endpoint**: Allowing to expose HTTP endpoint with different metrics in the Prometheus format
```

File: release-notes/release-6.15.0.md [hunk 3]
```diff
@@ -1,16 +1,39 @@
...
+Learn more in our [documentation](https://www.eclipse.org/che/docs/infra-support.html) and start using a shared Che server or local instance today.
+
+---
+
+# Eclipse Che 6.15 Release Notes
+
+Eclipse Che 6.15 includes:
+
+* **New CORS configuration parameters**: Allowing to test new CORS configuration before Che 6.16 release
+* **Prometheus metrics endpoint**: Allowing to expose http endpoint with different metrics in prometheus format
```
Comment (rkratky) [inline release-notes/release-6.15.0.md]:
```suggestion
* **Prometheus metrics endpoint**: Allowing to expose HTTP endpoint with different metrics in the Prometheus format
```

File: release-notes/release-6.16.0.md [hunk 4]
```diff
@@ -1,15 +1,20 @@
...
 * **Prometheus and Grafana deployed in the Helm chart**: Allowing to collect and inspect metrics emitted by the Che server
 * **Upgrade of Tomcat & CORS Configuration**: Changes to default CORS configuration and providing more configuration options
+* **Run Eclipse Che over https with a self-signed certificate**
 * **Improved REST API endpoint `/activity`**: It is now capable of returning information about
 workspace activity.
 
-## Upgrading
+---
+
+The Eclipse Che community is please to announce the 6.16 release, with a number of significant improvements. This release brings changes in the configuration of the handling of CORS (Cross-Origin Resource Sharing), and updates to the NodeJS and Java stacks on CentOS. This release contains a total of 61 enhancements or bug fixes.
```
Comment (rkratky) [inline release-notes/release-6.16.0.md]:
```suggestion
The Eclipse Che community is pleased to announce the 6.16 release with a number of significant improvements. This release brings changes in the configuration of the handling of CORS (Cross-Origin Resource Sharing) and updates to the NodeJS and Java stacks on CentOS. This release contains a total of 61 enhancements or bug fixes.
```

File: release-notes/release-6.16.0.md [hunk 5]
```diff
@@ -51,7 +56,21 @@ Here is the full list of environment variables that will be available in Che 6.1
 * `CHE_WSAGENT_CORS_ALLOWED__ORIGINS` = List of allowed origins in requests to WS Agent. If not set, or set to "null", value will be evaluated from CHE_API variable at runtime.
 * `CHE_WSAGENT_CORS_ALLOW__CREDENTIALS` = Allowing requests with credentials to WS Agent. Default is "true".
 
-PR with changes introduced in 6.16: https://github.com/eclipse/che/pull/12144
+**Pull requests:**
+* [Upgrade Tomcat and change default CORS configuration](https://github.com/eclipse/che/pull/12144)
+
+### Run Eclipse Che over https with a self-signed certificate
```
Comment (rkratky) [inline release-notes/release-6.16.0.md]:
```suggestion
### Run Eclipse Che over HTTPS with a self-signed certificate
```

File: release-notes/release-6.16.0.md [hunk 6]
```diff
@@ -51,7 +56,21 @@ Here is the full list of environment variables that will be available in Che 6.1
...
+
+![Che_on_https](https://www.eclipse.org/che/docs/images/workspaces/chrome_cert.png)
+
+Security is increasingly important for web services, and many Eclipse Che users would like to ensure communication between the Che server and client is encrypted and authenticated. In previous versions, it was not possible to start workspaces in Eclipse Che if your container platform was using a self-signed certificate. In this release, we have added that ability. The Che administrator can enable this feature by creating a secret with the certificate, and configuring the server to ensure that the certificate is used by the Che server.
+
+For more information, see https://www.eclipse.org/che/docs/che-6/openshift-config.html#https-mode---self-signed-certs
```
Comment (rkratky) [inline release-notes/release-6.16.0.md]:
```suggestion
For more information, see [HTTPS Mode - Self-Signed Certs](https://www.eclipse.org/che/docs/che-6/openshift-config.html#https-mode---self-signed-certs).
```

File: release-notes/release-6.16.0.md [hunk 7]
```diff
@@ -51,7 +56,21 @@ Here is the full list of environment variables that will be available in Che 6.1
...
+
+Links to PRs:
+[https://github.com/eclipse/che/pull/12089](https://github.com/eclipse/che/pull/12089)
```
Comment (rkratky) [inline release-notes/release-6.16.0.md]:
```suggestion
* [https://github.com/eclipse/che/pull/12089](https://github.com/eclipse/che/pull/12089)
```

File: release-notes/release-6.16.0.md [hunk 8]
```diff
@@ -51,7 +56,21 @@ Here is the full list of environment variables that will be available in Che 6.1
...
+[https://github.com/eclipse/che/pull/12112](https://github.com/eclipse/che/pull/12112)
```
Comment (rkratky) [inline release-notes/release-6.16.0.md]:
```suggestion
* [https://github.com/eclipse/che/pull/12112](https://github.com/eclipse/che/pull/12112)
```

File: release-notes/release-6.17.0.md [hunk 9]
```diff
@@ -23,16 +29,13 @@ failed workspaces. Server operators can see when workspaces are failing by using
...
 
-* Fixed issue’s title. (#ISSUE)
+**Pull requests**:
 
-## Community, Thank You!
-
-We’d like to say a big thank you to everyone who helped to make Che even better:
-
-* [Contributor Name](<PROFILE_URL>) – [Company Name](<COMPANY_URL>) – (#PR): [PR Title](<PR_URL>)
+Use new Factory and Containers plugins: https://github.com/eclipse/che/pull/12396
```
Comment (rkratky) [inline release-notes/release-6.17.0.md]:
```suggestion
* Use new Factory and Containers plugins: https://github.com/eclipse/che/pull/12396
```

File: release-notes/release-6.17.0.md [hunk 10]
```diff
@@ -23,16 +29,13 @@ failed workspaces. Server operators can see when workspaces are failing by using
...
+Factory support of devfile: https://github.com/eclipse/che/pull/12232
```
Comment (rkratky) [inline release-notes/release-6.17.0.md]:
```suggestion
* Factory support of devfile: https://github.com/eclipse/che/pull/12232
```

**Review by rkratky — APPROVED** [review]

**Review by slemeur — APPROVED** [review]

**Pull request merged**